Create ui-c8y-1021-37-4-realtime-operations-updates-filtered-as-expected.md (#2921)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-37-4-realtime-operations-updates-filtered-as-expected.md

---
date: ""

> title: Realtime operations updates are now filtered correctly

Reviewer: ```suggestion
title: Operation updates are now displayed in realtime in device control lists
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4323
version: 1021.37.4
---

> In Device Control lists operations with updated statuses remained displayed despite an active status filter. Now, operations that no longer match the selected filter are correctly removed from the list in real time.

Reviewer: ```suggestion
In device control lists, operations with an updated status remained displayed even if they did no longer match the active status filter criteria. Now, operations that no longer match the selected criteria are correctly removed from the list in realtime.
```

> In the past, realtime operations updates were not always filtered as expected, which could lead to incorrect or inconsistent information being displayed to users. This issue has now been resolved. With this change, realtime operations updates are properly filtered according to the defined criteria. Users will now see accurate and consistent realtime data in their operations views and reports, ensuring they have the correct information to make informed decisions.

Reviewer 2: ```suggestion
In Device Control lists operations with updated statuses remained displayed despite an active status filter. Now, operations that no longer match the selected filter are correctly removed from the list in real time.
```
